packages/ui/src/Calendar/Calendar.tsx

```diff
@@ -42,12 +48,12 @@ export const Calendar: FC<CalendarProps> = ({
     <DatePicker
       {...props}
       calendarClassName={cn(styles.calendar, calendarClassName)}
-      className={cn(styles.input, className)}
+      className={className}
       /*
        * Note: The library instantiates the component using React.cloneElement
        * Source: https://github.com/Hacker0x01/react-datepicker/blob/master/src/index.jsx#L926
        */
-      customInput={<CalendarInput label={inputLabel} />}
+      customInput={<CalendarInput label={inputLabel} textFieldSize={size} className={inputClassName} />}
```

Comment:
Seems like the icon is the same size
![image](https://user-images.githubusercontent.com/13898415/120195953-f8495280-c227-11eb-8705-05991486eca7.png)

Reply:
Hm yes, this one should probably be smaller. Let me fix that.

Reply:
Changed in https://github.com/hazelcast/hive/pull/279/commits/4e307d6d8048f2918e52ce3a1796230bd17efb77